README.md

```diff
@@ -12,20 +12,13 @@ SPDX-License-Identifier: GPL-3.0-or-later
 [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.proxmox)](https://codecov.io/gh/ansible-collections/community.proxmox)
 [![REUSE status](https://api.reuse.software/badge/github.com/ansible-collections/community.proxmox)](https://api.reuse.software/info/github.com/ansible-collections/community.proxmox)
 -->
-
-<!-- Describe the collection and why a user would want to use it. What does the collection do? -->
-
-<!-- Put your collection's mission statement in here. Example follows. -->
-<!--
 ## Our mission
 
-At the `community.proxmox` collection, our mission is to produce and maintain simple, flexible,
-and powerful open-source software tailored to `your collection purpose`.
+The Ansible Proxmox community team’s mission is to make automating Proxmox with Ansible nicer, better and more automated then ever!
```

Comment:
Either use a regular apostrophe here, or silence the ansible-test warnings (by adding ignores to `tests/sanity/ignore-2.xxx.txt`):
```suggestion
The Ansible Proxmox community team's mission is to make automating Proxmox with Ansible nicer, better and more automated then ever!
```
(BTW, you can also remove `tests/sanity/ignore-2.(15|16).*`, since these are no longer needed.)

Reply:
Oh right, that's probably from copy pasting from the group description on the forum :grimacing: